Update description about showing the Color Test Card. (#3796)
OttoWinter/esphomedocs: components/display/index.rst

Troubleshooting
---------------

Using the Color Test Card
-------------------------

If you're experiencing issues with your color display, the **`show_test_card: true`** option can help you to identify what might be wrong.
- It will show 3 color bars in **RED**, **GREEN** and **BLUE** to **BLACK^^ and the same 3 colors to **WHITE**. (1) 
- Together with that it will show the letters "**R**", "**G**" and "**B**" to validate the text oriantation.
- And as last it will show a rectangle around the corners of the display with a marker at the 0,0 corner most likely at the top left of the screen.

> When all points above are correctly shown correctly then everything is perfect and working as it should be.

Reviewer: ```suggestion
When all points above are shown correctly then the display is working as expected.
```
